src/interface/Options/ToggleFilters.tsx

```diff
@@ -8,10 +8,22 @@ export default () => {
   const { filtersShown, setFiltersShown } = useInterfaceContext();
   const toggleFilters = () => setFiltersShown(!filtersShown);
   const { t } = useTranslation('filter');
+  const label = filtersShown ? t('hide') : t('show');
 
   return (
-    <Button type="button" size="sm" onClick={toggleFilters} css={tw`whitespace-nowrap m-0`}>
-      {filtersShown ? t('hide') : t('show')}
+    <Button
+      role="switch"
+      aria-checked={filtersShown}
+      type="button"
+      size="sm"
+      onClick={toggleFilters}
+      css={tw`whitespace-nowrap m-0`}
+      id="toggle-filters"
+    >
+      {label}
+      <Label css={tw`sr-only`} htmlFor="toggle-filters">
+        Toggle Filters
```

Comment:
We should add a translation for this:
```suggestion
        {t('toggle-filters')}
```

Reply:
Submitted https://github.com/sajari/sdk-react/pull/799 👍